lib/github-service.js

```diff
@@ -29,6 +37,21 @@ class GithubService {
     });
   }
 
+  async createPullRequestComment(report){
+    getAssociatedPullRequestNumberFromSha(this.getOctokitInstance(), this.options.repo, this.options.refCommit)
```

Comment:
Return is missing here otherwise the promise chained by the then won't be returned